scripts/split-e2e-tests.ts

```diff
@@ -14,83 +14,100 @@ const CONCURRENCY = 35;
 // For now, this list is being used to skip creation of circleci jobs for these tasks
 
 // Todo: update the split test strategy to use parallelization so circleci results dont go over the limits of github payload size
-const WINDOWS_TEST_ALLOWLIST: string[] = [
-  'schema-function-1_pkg',
-  'tags_pkg',
-  'schema-auth-9-a_pkg',
-  'schema-auth-9-b_pkg',
-  'schema-auth-9-c_pkg',
-  'schema-model-a_pkg',
-  'schema-model-b_pkg',
-  'schema-model-c_pkg',
-  'schema-model-d_pkg',
-  'schema-model-e_pkg',
-  'api_lambda_auth_pkg',
-  'node-function_pkg',
-  'schema-function-2_pkg',
-  'notifications_pkg',
-  'interactions_pkg',
+const WINDOWS_TEST_SKIP_LIST: string[] = [
```

Comment:
PR looks good; is there a way we can see the before & after of this? how many new tests were added & did we lost any tests unintentionally?

Can you also do 1 or 2 E2E run with retries disabled?
- this should help us identify any flaky windows tests before we merge them into the dev branch
i believe this is configured in the local_publish_helpers file

Reply:
Good catch, I had skipped the `analytics.test.ts` file by mistake on Windows. The diffs should now be distinct. I've also run this test suite a number of times so CCI should have some good data already on non-deterministic Windows tests.